src/plugins/intel_cpu/src/nodes/executors/executor.hpp

```diff
@@ -112,18 +112,20 @@ class ExecutorContext {
           engine(graphContext->getEngine()),
           implPriorities(implPriorities),
           privateWeighCache(std::move(privateWeighCache)),
-          numNumaNodes(graphContext->getNumNumaNodes())
-    {}
+          numNumaNodes(graphContext->getNumNumaNodes()) {
+        auto cpuStreamsExecutor = graphContext->getCPUStreamExecutor();
+        curNumaNodeId = std::max(0, cpuStreamsExecutor ? cpuStreamsExecutor->get_numa_node_id() : curNumaNodeId);
+    }
 
     MultiCachePtr getRuntimeCache() const {
         auto runtimeCachePtr = runtimeCache.lock();
         assert(runtimeCachePtr);
         return runtimeCachePtr;
     }
 
-    DnnlScratchPadPtr getScratchPad(int subStreamID = 0) const {
+    DnnlScratchPadPtr getScratchPad(int subStreamID = -1) const {
```

Comment:
Can we make it interface more friendly for the user? 
I would expect call getScratchPad() (with default argument) to compute current Numa node ID internally. That will allow scratchpad user to avoid any calls like getCurNumaNodeId

Reply:
Ok, removed input parameter of getScratchPad().